lib/application/interfaces/Server.js

```diff
@@ -41,6 +41,25 @@ class Server {
     async listen() {
         return Promise.reject('The method or operation is not implemented.');
     }
+
+    /**
+     * Returns wether or not to accept any new incoming connections.
+     *
+     * @returns {Boolean} Wether or not to accept any new incoming connections.
+     */
+    shouldAcceptIncomingConnections() {
```

Comment:
sounds like a setter, without docs it is unclear, maybe:
```suggestion
    IsAcceptingIncomingConnections() {
```